### ISSUE TITLE
InputText freeze on focus loss

### ISSUE THREAD

**Issue by Azuna1** [ocornut/imgui#4972]
**Version/Branch of Dear ImGui:**

Version: 1.87
Branch: master

**Back-end/Renderer/Compiler/OS**

Back-ends: imgui_impl_win32.cpp + imgui_impl_dx9.cpp
Operating System: Windows 10

**My Issue/Question:**

Upgrading ImGui from version 1.80 to 1.87 introduced my application to deadlock in some "external code: Category IO | Kernel"
I was not able to pinpoint what actualy causes it.
Downgrading back to 1.80 resolves this issue

**Steps to reproduce:**
Click into demo widget inputtext, write text, click somewhere else inside or outside the imgui window

Cant reproduce on example_win32_directx9 v1.87

ImGui testcode was simply:

```
        ImGui_ImplDX9_NewFrame();
	ImGui_ImplWin32_NewFrame();
	ImGui::NewFrame();

	bool bopen = true;
	ImGui::ShowDemoWindow(&bopen);
	ImGui::EndFrame();
	ImGui::Render();
	ImGui_ImplDX9_RenderDrawData(ImGui::GetDrawData());
```

**Comment by ASTEYA49** [comment]
I am also experiencing the same issue.

**Comment by ocornut** [comment]
Sorry this has probably nothing to do with dear imgui and cannot be reproed in the demo.
Up to you to prove an issue but considering the crowd here I'll assume this is now our issue until proved wrong.

**Comment by Rex109** [comment]
I'm experiencing a similar problem, whenever i load a scene in my game the textbox unfocus/focus click has a random chance of freezing everything

**Comment by ocornut** [comment]
Reopening as we found a possible cause for this introduced in 1.86. Apologies for eagerly closing this.

**Comment by ocornut** [comment]
Pushed fix 133bbafa3cc7b7a1ca68808dacd1f6cee2552d4f

**Comment by ocornut** [comment]
Closing back in favor of tracking this in more-explicitly named #5535 - right now we disabled the call but we'll want to further investigate to restore the call.
